docs/source/documentation/2-docsearch-scraper/2-config-options.html.md.erb

```diff
@@ -152,113 +223,130 @@ then for each record the default value will be set
 
-### `allowed_domains`
+### `allowed_domains` _Optional_
 
 You can pass an array of strings. This is the whitelist of
 domains the crawler will browse. If a link targets a page that is not in the
 whitelist, the crawler will not follow it.
 
-Default is the domain of the first element in the `start_urls`
+Default is the domain of the first elements in the `start_urls`.
 
-### `stop_urls`
+### `min_indexed_level` _Optional_
 
-This is the blacklist of urls on which the crawler should stop. If a link in
-a crawled webpage targets one the elements in the `stop_urls` list, the crawler
-will not follow the link.
-
-Note that you can use regexps as well as plain urls.
-
-Note: It is sometimes needed to add `http://www.example.com/index.html` pages to
-the `stop_urls` list if you set `http://www.example.com` as a `start_urls`, to
-avoid duplicated content.
-
-### `selectors_exclude`
-
-By default, the `selectors` search is applied page-wide. If there are some parts
-of the page that you do not want to include (like a header, sidebar or footer),
-you can add them to the `selectors_exclude` key.
+Lets you define the minimum level at which you want a record to be indexed. For
+example, with a `min_indexed_level: 1`, you will only index records that have at
+least a `lvl1` field.
 
-### `custom_settings`
+This is especially useful when the documentation is split into several pages and
+all pages duplicate the main title or introduction (see [this issue][1]).
+With `min_indexed_level`, you can ignore the duplicated title.
 
-This object is any custom Algolia settings you would like to pass to the index
-settings.
+Default is `0`
 
-### `min_indexed_level`
+### `only_content_level` _Optional_
 
-Lets you define the minimum level at which you want records to be indexed. For
-example, with a `min_indexed_level: 1`, you will only index records that have at
-least a `lvl1` field.
+This attribute allows you to only index records which match the `text` selectors. Every other record will be skipped.
 
-This is especially useful when the documentation is split into several pages,
-but all pages duplicates the main title (see [this issue][1]).
+Default is `false`
 
-### `js_render`
+### `js_render` _Optional_
 
 The HTML code that we crawl is sometimes generated using Javascript. In those
 cases, the `js_render` option must be set to `true`. It will enable our
 internal proxy (Selenium) to render pages before crawling them.
 
-This parameter is optional and is set to `false` by default.
-
-### `js_wait`
+We highly recommend avoiding client-side rendering. It mainly decreases the performance of your website.
 
-The `js_wait` parameter lets you change the default waiting time to render the
-webpage with the Selenium proxy.
+Default is `false`
 
-This parameter is optional and is set to `0`s by default.
+### `js_wait` _Optional_
 
-### `use_anchors`
+The `js_wait` parameter lets you change the default waiting time (in seconds) to render the
```

Comment:
Not only for this one but I would mention dependency. In this case this option only makes sense when you have js_render: true